dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -73,10 +77,18 @@
 #   - https://endoflife.date/azure-kubernetes-service
 #   - https://endoflife.date/google-kubernetes-engine
 ALLOWED_KUBERNETES_VERSIONS = ["v1.23.13", "v1.24.7", "v1.25.3", "v1.26.0"]
+
 ALLOWED_EXECUTORS = ["KubernetesExecutor", "CeleryExecutor", "LocalExecutor", "CeleryKubernetesExecutor"]
+DEFAULT_ALLOWED_EXECUTORS = ALLOWED_EXECUTORS[2]
 ALLOWED_KIND_OPERATIONS = ["start", "stop", "restart", "status", "deploy", "test", "shell", "k9s"]
 ALLOWED_CONSTRAINTS_MODES_CI = ["constraints-source-providers", "constraints", "constraints-no-providers"]
 ALLOWED_CONSTRAINTS_MODES_PROD = ["constraints", "constraints-no-providers", "constraints-source-providers"]
+ALLOWED_EXECUTORS_FOR_A_BACKEND = {
+    MYSQL_BACKENDS: ALLOWED_EXECUTORS,
+    SQLITE_BACKENDS: [ALLOWED_EXECUTORS[2]],
+    POSTGRES_BACKENDS: ALLOWED_EXECUTORS,
+    MSSQL_BACKENDS: ALLOWED_EXECUTORS,
+}
```

Comment:
@potiuk Is this mapping correct?

Reply:
I think we do not need such backend if we remove sequential from the list and do the automated fallback for sqlite. Rather than such mapping it shoudl be simply "if sqlite => use sequential".

Reply:
Thanks @potiuk I'll change code accordingly